python/sparkts/utils.py

```diff
@@ -43,5 +44,7 @@ def datetime_to_nanos(dt):
         return pd.Timestamp(dt).value
     elif isinstance(dt, long):
         return dt
+    elif isinstance(dt, datetime):
+	return  long(dt.strftime("%s%f"))*1000
```

Comment:
Style nitpick: there should be spaces around the `*` and only one space before `long`